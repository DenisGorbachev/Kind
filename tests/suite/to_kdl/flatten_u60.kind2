A (b : B) (n : U60) : U60
A (B.new 2) 1 = 9

B : Type
B.new (n : U60) : B