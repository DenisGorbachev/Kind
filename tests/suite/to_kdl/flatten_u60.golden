fun (A0_ x0) {
  (A0_ #2) = #9
}

// A (b: (B)) (n: U60) : U60
fun (A b n) {
  (A {B_new .0} #1) = (A0_ .0)
}

// B.new (n: U60) : (B)
ctr {B_new n}

